utils/generate-catalog.sh

```diff
@@ -4,19 +4,14 @@
 
 set -euo pipefail
 
-### CONSTANTS
-# Used as well in the subscription object
-DEFAULT_CHANNEL=preview
-###
-
 OPM="${1?:Error \$OPM not set. Bye}"
 YQ="${2?:Error \$YQ not set. Bye}"
 BUNDLE_IMG="${3?:Error \$BUNDLE_IMG not set. Bye}"
 REPLACES_VERSION="${4?:Error \$REPLACES_VERSION not set. Bye}"
 LIMITADOR_OPERATOR_BUNDLE_IMG="${5?:Error \$LIMITADOR_OPERATOR_BUNDLE_IMG not set. Bye}"
 AUTHORINO_OPERATOR_BUNDLE_IMG="${6?:Error \$AUTHORINO_OPERATOR_BUNDLE_IMG not set. Bye}"
 DNS_OPERATOR_BUNDLE_IMG="${7?:Error \$DNS_OPERATOR_BUNDLE_IMG not set. Bye}"
-CHANNELS="${8:-$DEFAULT_CHANNEL}"
+CHANNELS="${8?:Error \$CHANNELS not set. Bye}"
```

Comment:
No channels, no catalog. We might need to review the release process of setting channel(s) and default channels

Reply:
the default channel is set at the makefile level. The script only requires one. I have moved the default from the script to the makefile. The script needs one channel value and fails if not provided. The default value is meaningful outside the script only.